vars/dockerExecute.groovy

```diff
@@ -1,51 +1,78 @@
 import com.cloudbees.groovy.cps.NonCPS
+import com.sap.piper.ConfigurationLoader
+import com.sap.piper.ConfigurationMerger
 
 def call(Map parameters = [:], body) {
 
     def STEP_NAME = 'dockerExecute'
```

Comment:
Use fields to have the name more readable at the top of the step, see https://github.com/SAP/jenkins-library/blob/master/vars/snykExecute.groovy#L7